tests/test_browser.py

```diff
@@ -73,6 +73,21 @@ def do_GET(s):
     httpd.handle_request()
 
 
+def with_wasmfs(f):
```

Comment:
I think other similar decorators are `also_with_*`, that is, with "also", which makes it clear we test the normal mode too.